src/include/daos_srv/pool.h

```diff
@@ -30,6 +31,12 @@
 /* Pool service (opaque) */
 struct ds_pool_svc;
 
+/* This is used to ensure in unit testing that clients ping targets after
+ * receiving a DER_RECONNECT error from the server in the case there is fault
+ * injection to return such an error.
+ */
+extern __thread bool firewall_blocking_flag;
```

Comment:
This can be removed.  It's no longer used because threadprivate support was added to FAIL_LOC